backend/lib/fetch/representative_points.py

```diff
@@ -1,28 +1,70 @@
 """Fetch representative points."""
+import json
+
+import logging
+
+from backend.config import config
 from backend.lib.database.postgres import connect
 from backend.lib.database.tables import representative_point, service_area
 from backend.lib.timer import timed
 
 from sqlalchemy.orm import sessionmaker
 
+logger = logging.getLogger(__name__)
+
+CENSUS_TABLES = ['census_acs_dp_02', 'census_acs_dp_03', 'census_acs_dp_04', 'census_acs_dp_05']
+CENSUS_MAPPING = json.load(open(config.get('census_mapping')))
+
+RP_COLUMNS = [
+    'id',
+    'representative_points.census_tract',
+    'county',
+    'latitude',
+    'longitude',
+    'population',
+    'service_area_id',
+    'zip_code'
+]
+
 
 @timed
 def fetch_representative_points(
     service_area_ids,
     format_response=True,
+    census_data=False,
     engine=connect.create_db_engine()
 ):
     """Fetch representative points for a list of service areas."""
     if not service_area_ids:
         return []
-
     id_list = '(' + ', '.join(["'%s'" % _id for _id in service_area_ids]) + ')'
-    select_query = 'SELECT * FROM {table_name} WHERE service_area_id in {id_list};'.format(
-        table_name=representative_point.RepresentativePoint.__tablename__,
+    if not census_data:
+        select_query = 'SELECT * FROM {table_name} WHERE service_area_id in {id_list};'.format(
+            table_name=representative_point.RepresentativePoint.__tablename__,
+            id_list=id_list)
+
+        representative_points = engine.execute(select_query)
+        return [
+            representative_point.row_to_dict(point, format_response)
+            for point in representative_points
+        ]
+
+    join_list = ' '.join([
+        'JOIN {table} ON (representative_points.census_tract = {table}.census_tract)'.format(
```

Comment:
IMO this should be a `LEFT JOIN` -- in case a census tract is missing, all the points should still be returned.